1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,6 +1,22 @@
-function setAlarm() {}
+function setAlarm() {
+const timeRemaining = document.getElementById("timeRemaining")
+const setAlarmInput = document.getElementById("alarmSet")
+let takingInput = Number(setAlarmInput.value)
+timeRemaining.innerText = "Time Remaining: " + takingInput + " Sec"
+
```

Comment:
I'm also able to enter negative numbers, can you think of a way to deal with those?